misc/hooks/protolint

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -euo pipefail
+set -uo pipefail
```

Comment:
The `e` makes the script to fail immediately, so the output from `go tool` was not printed and it was not obvious why it was not failing. Something llke this:
```
git ci -m "proto: adds multipooler admin gRPC api. Exposes methods related to replication and HA"
Linting go/pb/multipoolermanager go/pb/multipoolermanagerdata go/pb/pgctldata go/pb/pgctldservice
0 issues.
Prettier not found. Skipping...
```

Reply:
The example is for prettier -- was something similar happening with protolint? I would expect protolint to have better odds of success since it's not depending on any tooling other than `go tool`

Reply:
Hmm the example was for protolint, it was failing, but it doesn't tell you :D. After my change I do see the output. The prettier not found, doesn't fail right now, it just tell you.